delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/resources/ResourceRejection.scala

```diff
@@ -180,6 +181,12 @@ object ResourceRejection {
   final case class WrappedOrganizationRejection(rejection: OrganizationRejection)
       extends ResourceFetchRejection(rejection.reason)
 
+  /**
+    * Signals a rejection caused by a failure to indexing.
+    */
+  final case class WrappedIndexingActionRejection(rejection: IndexingActionFailed)
```

Comment:
Is the HTTP status code of this rejection handled somewhere? I guess by default should be BadRequest but worth it to double check.

Reply:
I think we've decided before that it should be InternalServerError, I'll fix that.